tests/test_missing.py

```diff
@@ -30,31 +30,37 @@ def _doit(testfile, **kwargs):
     result2 = active[0:2, 4:6, 7:9]
     np.testing.assert_array_equal(mean_result, result2["sum"]/result2["n"])
 
+@pytest.mark.xfail(USE_S3, reason="Missing data not supported in S3 yet")
 def test_partially_missing_data(tmp_path):
     testfile = str(tmp_path / 'test_partially_missing_data.nc')
     r = dd.make_partially_missing_ncdata(testfile)
     _doit(testfile, missing_value=-999.)
 
+@pytest.mark.xfail(USE_S3, reason="Missing data not supported in S3 yet")
 def test_missing(tmp_path):
     testfile = str(tmp_path / 'test_missing.nc')
     r = dd.make_partially_missing_ncdata(testfile)
     _doit(testfile, missing_value=-999.)
 
+# FIXME: why does it pass for S3?
 def test_fillvalue(tmp_path):
     testfile = str(tmp_path / 'test_fillvalue.nc')
     r = dd.make_fillvalue_ncdata(testfile)
     _doit(testfile, _FillValue=-999.)
 
+# FIXME: why does it pass for S3?
 def test_validmin(tmp_path):
     testfile = str(tmp_path / 'test_validmin.nc')
     r = dd.make_validmin_ncdata(testfile)
     _doit(testfile, valid_min=-1.)
 
+# FIXME: why does it pass for S3?
```

Comment:
remember these tests are run on POSIX, no S3 involved here

Reply:
In the normal test run they run on POSIX. In the S3/Minio test run they run on S3.

Reply:
On L20,  `uri = utils.write_to_storage(testfile)`

Reply:
indeed but USE_S3 is not set to True because of two things:
- you are exporting the var `USE_S3 = True` to config.py in the GA workflow but that config is created ad hoc and not imported as the module we have (I should have asked you this yesterday at PR review point, but I missed it, my apologies): `echo 'USE_S3 = True\nPROXY_URL = "http://localhost:8080"' >> config.py` if you want that exported and written in our module that needs be `activestorage/config.py`; if you want that as an env var that needs be exported to the `GITHUB_ENV`
- there is still a legacy check inside `active.py` L388 that requires an `storage_type` kwarg to `Active` class be set to s3 specifically, that we'll have to either get rid or use it correctly

At any rate, let me do it this way - if you don't mind, I'm gonna close these PRs and let's do it this way from now on, given that's quite a few of us working on this project already, and it's starting to be hard to keep track of what PR does what and why:

- let's first open an issue with the thing that we want to fix, discuss there if the fix is really needed, and if so, how to fix it (fix could be feature too :grin: )
- I will plug in a minimal checkbox-based PR template - so we have adescriptive title, minimal description of the PR and what it does, and Closes # the issue number
- and we'll take it from there :smiley: 

attn: @bnlawrence who is a big fan of issue first, PR after too :beer: